sherpa/astro/data.py

```diff
@@ -3910,6 +3947,18 @@ def notice(self, lo=None, hi=None, ignore=False, bkg_id=None):
         such as a channel range of 2000 to 3000 when the valid range
         is 1 to 1024 - then all points will be ignored.
 
+        When filtering with channel units then:
+
+        - the lo and hi arguments, if set, must be integers,
+        - and the lo and hi values are inclusive.
+
+        For energy and wavelength filters:
+
+        - the lo and hi arguments, if set, must be >= 0,
+        - and the lo limit is inclusive but the hi limit is exclusive.
+
+        For
```

Comment:
```suggestion
```

I have no idea what this was meant to be. I;ll hold off fixing until the code has been reviewed.